specs/project_initialization.spec

```
@@ -52,12 +52,10 @@ Initializing unknown language project
 tags: java, csharp, dotnet, ruby, python, js
 
 * In an empty directory, use default initialization of a project named "init_proj_unknown" in language "unknown"
-* Console should contain following lines in order 
+* Console should contain following lines in order
 
-   |console output                                                       |
-   |---------------------------------------------------------------------|
-   |Compatible language plugin unknown is not installed.                 |
-   |Failed to install plugin 'unknown'.                                  |
-   |Reason: Invalid plugin. Could not download unknown-install.json file.|
+   |console output                                                                     |
+   |-----------------------------------------------------------------------------------|
+   |failed to initialize project. cannot find a Gauge template 'unknown'     |
```

Shouldn't this have case fixes.

```suggestion
   |Failed to initialize project. Cannot find a Gauge template 'unknown'     |
```